mypy/stubgen.py

```diff
@@ -623,7 +634,7 @@ def __init__(
         # Disable implicit exports of package-internal imports?
         self.export_less = export_less
         # Add imports that could be implicitly generated
-        self.import_tracker.add_import_from("typing", [("NamedTuple", None)])
+        self.import_tracker.add_import_from("typing", [("NamedTuple", None), ("TypedDict", None)])
```

Comment:
In typeshed we always import `TypedDict` from `typing_extensions` rather than `typing`, as typeshed/mypy still support Python 3.7, and TypedDict was added to `typing` in Python 3.8 (but is available from `typing_extensions` on all supported Python versions)

Reply:
Ooh good point. I'll change it

Reply:
Looking at it, this should not be treated as "implicitly generated". Unlike named tuples where a `collections.namedtuple` gets implicitly transformed to a `typing.NamedTuple`, typed dicts are never generated from other forms. I'll add it to known imports though so that it always imports from `typing_extensions`:
https://github.com/python/mypy/blob/2e75cbaa8e5c0fb8aa3548d5c7c8ccfd00131057/mypy/stubgen.py#L640-L644